src/Analyzer/CodeAnalyzer.cs

```diff
@@ -89,7 +91,8 @@ private void Dispose(bool disposing)
             }
 
             // delete created temporary directory
-            Directory.Delete(tmpSonarLintFolder, true);
+            if(tmpSonarLintFolder != null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/Codacy/codacy-sonar-csharp/pullRequest?prid=4528542)